context/auth.tsx

```diff
@@ -1,16 +1,44 @@
 import React from "react"
 import { loadFirebase } from "./firebase"
 import axios from "../util/axios"
+import firebase from "firebase"
```

Comment:
This import name is conflicting with the variable names in this module

Reply:
done